Semantics.md

```diff
@@ -174,6 +174,12 @@ the addition of the offset to the address never causes wrapping, so if the
 address for an access is out-of-bounds, the effective address will always also
 be out-of-bounds.
 
+It is a validation error if a load or store has an offset immediate where:
+```
+offset + numBytesTouched(load/store) - 1 > 2^32 - 1
+```
+If this condition is true, the load or store is guaranteed to be out of bounds.
```

Comment:
Both of my comments apply here as well.

Alternatively, instead of having two copies of this rule, it might be sufficient to just state it here in Semantics.md and omit it in BinaryFormat.md.